.changeset/rare-deers-attend.md

```diff
@@ -0,0 +1,13 @@
+---
+"@razorpay/blade": patch
```

Comment:
Should be patch or minor?

Reply:
@chaitanyadeorukhkar I feel it should be minor since it is a feature.